src/components/bookList/BookList.tsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from 'react'
+
+import InfiniteScroll from 'react-infinite-scroll-component'
+
+import { Book } from '../../model/Book'
+import BookRequest from '../../model/BookRequest'
+import BookService from '../../services/BookService'
+import Card from '../card/Card'
+import './BookList.css'
+
+
+const BookList = () => {
+  const [ book, setBook ] = useState<Book[]>([])
+  const [ hasMore, setHasMore ] = useState(true)
+  const [ pageNubmer, setPageNumber ] = useState(1)
+
+  const numberOfCard = 24
```

Comment:
You can define constants like this one outside of your component, for example on line 11, no need to define them here

Reply:
OK, I move it outside.